<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuestionWindow</class>
 <widget class="QDialog" name="QuestionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>554</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>请编辑问题...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>40</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <widget class="QTextEdit" name="txtedtQues">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="placeholderText">
          <string>请输入问题...</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>A</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="edtQptionA">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>40</height>
              </size>
             </property>
             <property name="placeholderText">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>B</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="edtQptionB">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>40</height>
              </size>
             </property>
             <property name="placeholderText">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>C</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="edtQptionC">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>40</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>D</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="edtQptionD">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>40</height>
              </size>
             </property>
             <property name="placeholderText">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>发送至：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblSendToName">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>全班同学</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QTextEdit" name="txtedtAnswer">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>学生的回答...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="tableView"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>20</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnSend">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>发送</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnCount">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>统计</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnQuit">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>退出</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
